docker_dyno_driver.go

```diff
@@ -120,6 +133,33 @@ func (dd *DockerDynoDriver) Stop(ex *Executor) error {
 	return dd.d.c.StopContainer(ex.container.ID, 10)
 }
 
+func (dd *DockerDynoDriver) assignAddressAndPort(ex *Executor) error {
+	timeout := time.After(ContainerInspectTimeout)
+	for {
+		select {
+		case <-timeout:
+			return ErrContainerInspectTimeout
+		default:
+			ctr, err := dd.d.c.InspectContainer(ex.container.ID)
```

Comment:
isn't this spinning and burning the CPU cycles? Would it be better to only run it every Xs to avoid busy waiting here? `time.NewTicker` could help, something like this:

https://play.golang.org/p/xB0r36oBsX

Uncomment the `time.Sleep(3)` line if you want to see the (expected) behavior when a job takes more than a "tick" duration to complete.